sklearn/linear_model/_ridge.py

```diff
@@ -2149,7 +2147,7 @@ def fit(self, X, y, sample_weight=None, score_params=None):
 
         for i, alpha in enumerate(np.atleast_1d(self.alphas)):
             G_inverse_diag, c = solve(float(alpha), y, sqrt_sw, X_mean, *decomposition)
-            if scorer is None:
+            if self.scoring is None:
```

Comment:
Oh this was being called again redundantly?

Reply:
Yes, there was this `scorer = self._get_scorer()` line above, and I figured it would be easier to read if we re-uses what we have passed anyways.